en_US/installation/0-setup.dns.md

```diff
@@ -123,9 +123,9 @@ be delivered to server `mail.mydomain.com`.
 
 ### What is an autoconfig/autodiscover record
 
-autoconfig/autodiscover.company.com records allow mail clients to fetch automatically the mail
-client configuration of a mailbox. If the mailbox to configure is user@company.com then
-it will automatically check autodiscover.company.com for the correct configuration.
+`autoconfig/autodiscover.company.com` records allow mail clients to fetch automatically the mail
```

Comment:
Please help change `autoconfig/autodiscover.company.com` to "`autoconfig.company.com` and `autodiscover.company.com`" to avoid confusion.